core/src/messages/dns.rs

```diff
@@ -20,17 +27,32 @@ pub struct SetDnsRecord {
 }
 
 impl TypedMessage for SetDnsRecord {
-    type Response = bool;
+    type Response = NoReply;
 
     fn subject(&self) -> String {
         format!(
             "cluster.{}.dns.{}",
-            self.cluster,
+            self.cluster.subject_name(),
             serde_json::to_string(&self.kind).unwrap()
         )
     }
 }
 
+impl JetStreamable for SetDnsRecord {
+    fn config() -> async_nats::jetstream::stream::Config {
+        async_nats::jetstream::stream::Config {
+            name: Self::stream_name().into(),
+            subjects: vec!["cluster.*.dns.*".into()],
+            max_age: Duration::from_secs(60),
```

Comment:
this 60 is the minimum rate at which this message can be emitted without periods in which no dns is set, correct? if so, we should have a way to make sure we don't accidentally cause that, or a test to check for it.

Reply:
I will defer this to #187